theme/open/views/dataprivacy/privacy_tom.hbs

```diff
@@ -0,0 +1,6 @@
+<h3>
+	Technische und organisatorische Maßnahmen
+</h3>
+<p>
+	Die technischen und organisatorischen Maßnahmen für die {{theme.title}} finden Sie hier: <a href="https://s3.hidrive.strato.com/schul-cloud-hpi/open/Dokumente/TOMs.pdf" target="_blank" rel="noopener">hier</a>.
+</p>
```

Comment:
I think we do not need this file.
`{{@root.theme.documents.baseDir}}Dokumente/TOMs.pdf` like in the default handlebars file works here as well.
It works, because the theme name is automaticly added to the `baseDir` ( https://github.com/schul-cloud/schulcloud-client/blob/develop/config/documents.js#L29 )